### ISSUE TITLE
State changes ignored by component

### ISSUE THREAD

**Issue by kuatro** [TriPSs/react-esc#19]
Hey @TriPSs . Me again.
I found, that components aren't reacting to state changes.

```
@resolve('jsonPlaceholder', (props) => props.fetchData(props.params.id).then(action => action.payload), false)
export class Async extends React.Component {
  componentWillReceiveProps (nextProps) {
    if (nextProps.params.id !== this.props.params.id) {
      this.props.fetchData(nextProps.params.id)
    }
  }
}
```

After data fetched and stored properly, the component shows previous state anyway.

![2017-03-03 16 09 38](https://cloud.githubusercontent.com/assets/5260027/23546912/d54ba41c-002b-11e7-96e9-db31f149923d.png)

**Comment by kuatro** [comment]
#20

**Comment by TriPSs** [comment]
Fixed in [v1.4.21](https://github.com/TriPSs/react-esc/releases/tag/v1.4.21)
